modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_aggregation/angle_aggregation.h

```diff
@@ -26,38 +26,43 @@ namespace lbs
  * The octant based separation is achieved via the notion of "AngleSetGroup"
  * which will group angle sets for each quadrant or octant
  * (depending on 2D or 3D).*/
+
 class AngleAggregation
 {
-public:
-  std::vector<AngleSetGroup> angle_set_groups;
-  std::map<uint64_t, std::shared_ptr<SweepBndry>> sim_boundaries;
-  size_t number_of_groups = 0;
-  size_t number_of_group_subsets = 0;
-  std::shared_ptr<AngularQuadrature> quadrature = nullptr;
-
 private:
-  bool is_setup = false;
-  std::pair<size_t, size_t> number_angular_unknowns;
-  bool num_ang_unknowns_avail = false;
+  bool is_setup_;
+  size_t number_of_groups_;
+  size_t number_of_group_subsets_;
+  bool num_ang_unknowns_avail_;
+  std::pair<size_t, size_t> number_angular_unknowns_;
```

Comment:
Could you unify this so that the variable names are `num_` instead of `number_`? 
From the code `num_` vs. `number_` is ~4000 vs. ~100. 
This could be a separate patch.

Reply:
Fixed.